change link checker update job to run 15 hours after midnight

jsjiang opened the pull request:
@sfisher Hi Scott, this is to schedule the link checker update to restart at 3pm (15 hours after midnight). See lines 545-548 in the setting.py.j2 file for the explanation. The value is used in the [`proc-link-chekcer.py` script to calculate time to sleep before next run](https://github.com/CDLUC3/ezid/blob/main/ezidapp/management/commands/proc-link-checker-update.py#L122-L128).

I have deployed tag `link-checker-update` on ezid-stg with this setting and the `proc-link-chekcer` job was started at 3pm as expected.

Please take a look and let me know if you have questions.

Thank you

Jing

sfisher reviewed (approved):
This looks good to me, thanks for explaining it all to me, Jing.

jsjiang commented:
@sfisher Hi Scott,
The link-checker-update job was restarted by itself today at 3pm. So the setting worked. I changed the start time from 15 hours after midnight to 18 hours (64,800 sec) to avoid a conflict with Prod OS patching which happens once a month on the first Tuesday 3-5pm. 

Also I changed a filename in the `tests/util` folder so it will not be picked up by `pytest`.

I will merge the change and I am planning to run prod deploy Monday 9/16 in the morning.

Thank you

Jing

sfisher commented:
Thank you, Jing!
